chapter_10/10-8.py

```diff
@@ -0,0 +1,38 @@
+# 10-8. Cats and Dogs
+
+from pathlib import Path
+
+# deal with "What folder am I working from?"
+import os
```

Comment:
Should this be with the above import statement rather than down here with figuring out the folder?